test/kinetics/kineticsFromYaml.cpp

```diff
@@ -10,7 +10,26 @@
 
 using namespace Cantera;
 
-TEST(Reaction, ElementaryFromYaml)
+TEST(ReactionRate, ModifyArrheniusRate)
+{
+    auto sol = newSolution("gri30.yaml");
```

Comment:
To avoid unnecessarily creating a `MixTransport` object, use
```suggestion
    auto sol = newSolution("gri30.yaml", "", "None");
```